x-pack/test/apm_api_integration/tests/asset_services/asset_services.spec.ts

```diff
@@ -185,7 +185,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       const tracesGenAssets = createGeneratorFromArray(traces);
 
       return Promise.all([
-        assetsSynthtraceClient.index(
+        entitiesSynthtraceClient.index(
           Readable.from(Array.from(logsGenAssets).concat(Array.from(tracesGenAssets)))
         ),
         logSynthtraceClient.index(logsGen),
```

Comment:
Should we rename these too?
```suggestion
        entitiesSynthtraceClient.index(
          Readable.from(Array.from(logsGenEntities).concat(Array.from(tracesGenEntites)))
        ),
        logSynthtraceClient.index(logsGen),
```